grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -137,11 +138,27 @@ class GrabbitConfiguration {
 
     @CompileStatic
     static class PathConfiguration {
+        private static final String COMMA = ","
+        private static final String SLASH = "/"
+        private static final String EMPTY = ""
         String path
+        Collection<String> excludePaths
         Collection<String> workflowConfigIds
+        
+        Collection<String> modifyExcludePaths(@Nonnull Collection<String> excludePaths)
```

Comment:
An instance method called "modify*" implies that it will modify the instance, which this doesn't do. It should be made `static` and renamed to say what it actually does (`toAbsolutePaths` or something).